library/build.gradle

```diff
@@ -40,7 +40,7 @@ android {
 }
 
 dependencies {
-    implementation "org.jetbrains.kotlin:kotlin-stdlib:$kotlinVersion"
+    api "org.jetbrains.kotlin:kotlin-stdlib:$kotlinVersion"
```

Comment:
How Kotlin is exposed?

Reply:
This is one I went on the side of caution for. The only place I could see that might have potential impact was `ChuckerInterceptor` where the set of headers to redact is exposed. I don't have a java project to easily test this with.

In practice, this probably isn't an issue.

Reply:
Yes, there is no need for that. `kotlin.collections.Set` is seen as `java.util.Set` from Java.